.github/workflows/prebuild.yml

```diff
@@ -0,0 +1,44 @@
+on:
+  push:
+    tags:
+      - 'v*'
+
+name: Prebuild
+
+jobs:
+  prebuild-node:
+    name: Upload Node Prebuild
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [macos-latest, windows-latest, ubuntu-latest]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: 14
+      - run: npm install
+      - name: build release
+        run: |
+          npx prebuild -r node -t 10.20.0 -t 12.0.0 -t 14.0.0 --include-regex 'better_sqlite3.node$' -u ${{ secrets.GITHUB_TOKEN }}
+
+  prebuild-electron:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        # os: [windows-latest]
+        os: [macos-latest, windows-latest, ubuntu-latest]
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2-beta
+        with:
+          node-version: '14'
+      - run: npm install
+      - run: npm run test
+      - run: npm run build-release
+      - name: create prebuild
+        run: |
+          npx prebuild -r electron -t 7.0.0 -t 8.0.0 -t 9.0.0 --include-regex 'better_sqlite3.node$'  -u ${{ secrets.GITHUB_TOKEN }}
+        # npx prebuild -r electron -t 9.0.0 --include-regex 'better_sqlite3.node$' --arch x86 -u ${{ secrets.GITHUB_TOKEN }}
```

Comment:
What's this commented line for?

Reply:
when I wrote this for my project I was using only electron 9, so that commented one is the one used in my case. that comment can be removed as it doesn't have any relevance, and the supported election version to be added to be updated in the line 43.